docs/content/documentation/deployment/gitlab-pages.md

```diff
@@ -3,68 +3,91 @@ title = "GitLab Pages"
+It is recommended to create a repository on GitLab that contains solely your
+Zola project. The [Zola's directory structure](https://www.getzola.org/documentation/getting-started/directory-structure/)
+should be located in the root directory of your repository.
 
-Under your group `websites`, you created a project called `websites.gitlab.io`. Your project’s URL will be `https://gitlab.com/websites/websites.gitlab.io`. Once you enable GitLab Pages for your project, your website will be published under `https://websites.gitlab.io`.
+For information on how to create and manage a repository on GitLab, refer to:
 
+<https://docs.gitlab.com/ee/user/project/repository/>
 
-This guide assumes that your Zola project is located in the root of your repository.
+## Ensuring that the runner can access your theme
 
-## Ensuring that the CI runner can access your theme
-
-Depending on how you added your theme, your repository may not contain it. The best way to ensure that the theme will
-be added is to use submodules. When doing this, ensure that you are using the `https` version of the URL.
+Depending on how you added your theme, your repository may not contain it.
+The best way to ensure that the theme will be added is to use submodules.
+When doing this, ensure that you are using the `https` version of the URL.
 
 ```bash
-$ git submodule add {THEME_URL} themes/{THEME_NAME}
+git submodule add {THEME_URL} themes/{THEME_NAME}
 ```
 
 For example, this could look like:
 
 ```bash
-$ git submodule add https://github.com/getzola/hyde.git themes/hyde
+git submodule add https://github.com/getzola/hyde.git themes/hyde
 ```
 
-## Setting up the GitLab CI/CD Runner
+## Setting up the GitLab Runner
 
-The second step is to tell the GitLab continuous integration runner how to create the GitLab page.
+The GitLab Runner needs to know how to create your site in order to deploy
+it to the GitLab Pages server.
 
-To do this, create a file called `.gitlab-ci.yml` in the root directory of your repository.
+We provide you with a template to accomplish this task easily.
+Create a file called `.gitlab-ci.yml` in the root directory of your
+repository and copy the contents of the template below.
 
 ```yaml
-image: alpine:latest
+stages:
+  - deploy
+
+default:
+  image: alpine:latest
+  tags:
+    - docker
+
 variables:
-  # This variable will ensure that the CI runner pulls in your theme from the submodule
-  GIT_SUBMODULE_STRATEGY: recursive
+  # The runner will be able to pull your Zola theme when the strategy is
+  # set to "recursive".
+  GIT_SUBMODULE_STRATEGY: "recursive"
 
 pages:
+  stage: deploy
   script:
-    # Install the zola package from the alpine community repositories
-    - apk add --update-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ zola
-    # Execute zola build
+    - apk add zola
```

Comment:
Is it possible to specify a version?

Reply:
I have modified the file to allow setting a specific version of Zola.

Reply:
Note that Alpine only has the latest versions of packages (binary builts) and that the `zola` executable does not run well, so I switched to Debian and downloading the compiled executables from GitHub releases.